package.json

```diff
@@ -204,7 +206,8 @@
 	},
 	"overrides": {
 		"react": "^18.2.0",
-		"react-dom": "^18.2.0"
+		"react-dom": "^18.2.0",
+		"next-mdx-remote": "^4.2.0"
```

Comment:
Is this override for `swingset`'s internal next-mdx-remote?

Reply:
This is mainly to satisfy npm, as it warns on each install that `react-docs-page`'s `next-mdx-remote` peer dependency is not met (as it relies on `3.x`)